### PR TITLE
Fix bounding calculation of tag-list and more button

### PR DESCRIPTION
**Pull request by rowa-audil**
Fixes bounding calculation for the tag-list component.

(new branch because git didn't let me commit to the old branch)

### CODE REVIEW

File: components/tag/src/tag-list/tag-list.ts [hunk 1]
```diff
@@ -69,6 +69,9 @@ export class DtTagList implements AfterContentInit, OnDestroy {
   /** @internal Reference to wrapper directives */
   @ViewChild('wrapper', { static: true }) _wrapperTagList: ElementRef;
 
+  /** @internal Reference to more button */
+  @ViewChild('moreBtn', { static: false }) _moreBtn: ElementRef;
```
Comment (tomheller) [inline components/tag/src/tag-list/tag-list.ts]: Do you actually need the moreBtn? I cannot find a reference to it anywhere.

**Comment by tomheller** [comment]
Code looks good, please rebase and squash after the review.

**Comment by rowa-audil** [comment]
@tomheller Please have a look again. I rebased my commits

**Comment by tomheller** [comment]
Resolved conflicting files, should be good now!

**Pull request merged**